keps/sig-apps/592-ttl-after-finish/README.md

```diff
@@ -427,3 +427,6 @@ When a Pod is created or updated:
   - indicate that the feature will be graduated for Jobs, and that Pods will be done as future work under a separate flag
   - add production readiness questionnaire
   - mark the feature for Beta graduation for jobs.
+- 2021-07-27: KEP updated to
+  - indicate that the feature will be graduated to stable for Jobs
+  - Pods will be done as future work if the need arises
```

Comment:
should I remove all pod references in this KEP?

Reply:
I think, you can leave the pod references.